### PR TITLE
Initial drive to distance pid

### PR DESCRIPTION
**Pull request by Tigerbytes1775**

### CODE REVIEW

File: src/org/usfirst/frc/team1775/robot/OI.java [hunk 1]
```diff
@@ -42,7 +43,7 @@
 	public static Joystick myJoystick;
 	public void init () {
 	    myJoystick = new Joystick(0);
```
Comment (clebio) [inline src/org/usfirst/frc/team1775/robot/OI.java]: What about *my* joystick?

File: src/org/usfirst/frc/team1775/robot/RobotMap.java [hunk 2]
```diff
@@ -31,6 +33,10 @@ public static void init() {
 	    motorController = new Talon(1);
 	    motorController2 = new Talon(2);
 	    drive = new DifferentialDrive(motorController, motorController2);
-    }
+    
+	    driveEncoder = new Encoder(3, 2, false, Encoder.EncodingType.k1X);
+		double distancePerPulse = ((3.19*Math.PI)/250.0);
```
Comment (clebio) [inline src/org/usfirst/frc/team1775/robot/RobotMap.java]: Magic numbers!

**Review by clebio — APPROVED** [review]
just some minor nits

**Review by augustus-thomas — APPROVED** [review]
Remove magical numbers over time. Everything else seems fair

**Review by augustus-thomas — APPROVED** [review]

**Pull request merged**